etp-backend/src/main/clj/solita/etp/api/energiatodistus.clj

```diff
@@ -32,6 +32,8 @@
                         {:type :invalid-arguments :response 400}
                         {:type :schema.core/error :response 400}])
 
+(def et-get-access (some-fn rooli-service/laatija? rooli-service/paakayttaja? rooli-service/laskuttaja?))
```

Comment:
Tämä on nyt myös siellä crud-moduulissa. En vielä yhdistellyt kun en ollut varma, mihin se sitten kuuluisi ja halutaanko ylipäänsä näitä toistuvia accesseja koota vakioksi. Jos on toiveita että miten näille tehdään, niin mieluusti kuulen niistä.

Reply:
ehkä tämän voisi laittaa rooli-serviceen esim. nimellä et-reader? vähän samaan tyyliin kuin laatija-reader?

Reply:
nyt pitäisi olla tämän ehdotuksen mukainen